recipes/example/meta.yaml

```diff
@@ -1,82 +1,58 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
You should **not** be modifying the `recipes/example/meta.yaml` file. Please only modify the recipe for the package you're trying to add in `recipes/nnetsauce/meta.yaml`.